src/lib/IONOS/components/IONOS_Footer.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+	const currentYear = new Date().getFullYear();
+</script>
+
+<footer>
```

Comment:
the footer has following "surprises"
* z-index problem 
![image](https://github.com/user-attachments/assets/0bf8d6ab-bd93-47a0-b431-f1dc2691e511)
* positioning on different screen sizes 
![image](https://github.com/user-attachments/assets/8027cf6a-8163-4299-bd25-1d568f159083)